src/DataViews/PresetsDataViewTest.cpp

```diff
@@ -206,18 +207,18 @@ TEST_F(PresetsDataViewTest, CheckPresenceOfContextMenuEntries) {
   view_.OnSort(1, DataView::SortingOrder::kAscending);
 
   // Loadable preset
-  EXPECT_THAT(view_.GetContextMenu(0, {0}),
+  EXPECT_THAT(FlattenContextMenuWithGrouping(view_.GetContextMenuWithGrouping(0, {0})),
               testing::UnorderedElementsAre("Copy Selection", "Export to CSV", "Load Preset",
```

Comment:
Do we have a test to check consistency between order in different DataViews? 

Ideally I would like to check that for every pair A,B, if both are present in Data_view_1 and Data_view_2, the order is the same. But that could be really complex and include different DataViews. To make it Unit-testable, we can have a strict defined order between actions already predefined and check for each DataView that the order holds for them. What do you think?

Reply:
Make sense.  I will think about how to do this.